docs/src/main/sphinx/release/release-452.md

```diff
@@ -0,0 +1,82 @@
+# Release 452 (11 Jul 2024)
+
+## General
+
+* Add support for processing the `X-Forwarded-Prefix` header when the
+  `http-server.process-forwarded` property is enabled. ({issue}`22227`)
+* Add support for the {func}`euclidean_distance`, {func}`dot_product`, and
+  {func}`cosine_distance` functions. ({issue}`22397`)
+* Improve performance of queries with selective joins by performing fine-grained
+  filtering of rows using dynamic filters. This behavior is enabled by default
+  and can be disabled using the `enable-dynamic-row-filtering` configuration
+  property or the `enable_dynamic_row_filtering` session property. ({issue}`22411`)
+* Fix a performance regression when using the native filesystem for Azure. ({issue}`22561`)
+* Fix sporadic query failure when the `retry_policy` property is set to `TASK`. ({issue}`22617`)
+
+## Web UI
+
+* Fix query plans occasionally not rendering the stage details page. ({issue}`22542`)
+
+## BigQuery connector
+
+* Add support for using the
+  [BigQuery Storage Read API](https://cloud.google.com/bigquery/docs/reference/storage)
+  when using the [`query` table function](bigquery-query-function). ({issue}`22432`)
+
+## Black Hole connector
+
+* Add support for adding, dropping and renaming columns. ({issue}`22620`)
+
+## ClickHouse connector
+
+* Add [`query` table function](clickhouse-query-function) for full query
+  pass-through to ClickHouse. ({issue}`16182`)
+
+## Delta Lake connector
+
+* Add support for type coercion when adding new columns. ({issue}`19708`)
+* Improve performance of reading from Parquet files with large schemas. ({issue}`22434`)
+* Fix incorrect results when reading `INT32` values in Parquet files. ({issue}`21556`)
+
+## Hive connector
+
+* Add support for coercion of structural data types for non-partitioned tables
+  using ORC files. ({issue}`22326`)
+* Add support for type coercion when adding new columns. ({issue}`19708`)
+* Add support for `varbinary` to `varchar` type coercion. ({issue}`22322`)
+* Improve performance of reading from Parquet files with large schemas. ({issue}`22434`)
+* Fix incorrect results when reading `INT32` values in Parquet files. ({issue}`21556`)
```

Comment:
It's to varchar and decimal, I think.

Reply:
It might be good to call out, as users may not know what type the files contain but they know what types they deal with on the Trino side. That will call their attention in case they are using those types and have seen problems in the past.